source/features/highlight-collaborators-and-own-conversations.tsx

```diff
@@ -22,7 +22,10 @@ const collaborators = new CachedFunction('repo-collaborators', {
 
 async function highlightCollaborators(signal: AbortSignal): Promise<void> {
 	const list = await collaborators.get();
-	observe('.js-issue-row [data-hovercard-type="user"]', author => {
+	observe([
+		'.js-issue-row [data-hovercard-type="user"]',
+		'[data-testid="list-row-repo-name-and-number"] a[class^="issue-item-module__authorCreatedLink"]',
```

Comment:
I think the class name is pretty specific already 
```suggestion
		'a[class^="issue-item-module__authorCreatedLink"]',
```